src/bootstrap/flags.rs

```diff
@@ -433,13 +433,7 @@ Arguments:
         // Get any optional paths which occur after the subcommand
         let paths = matches.free[1..].iter().map(|p| p.into()).collect::<Vec<PathBuf>>();
 
-        let cfg_file = matches.opt_str("config").map(PathBuf::from).or_else(|| {
-            if fs::metadata("config.toml").is_ok() {
-                Some(PathBuf::from("config.toml"))
-            } else {
-                None
-            }
-        });
+        let cfg_file = env::var("BOOTSTRAP_CONFIG").ok().map(PathBuf::from);
```

Comment:
Mind using env::var_os here? Technically I doubt we'll never to_str it but seems good to do the right thing regardless.

Reply:
Fixed, thanks!